integration/tests/access/unstaked_node_test.go

```diff
@@ -121,32 +156,57 @@ func (suite *UnstakedAccessSuite) TestReceiveBlocks() {
 	// 3. Check that staked AN also processed the block. This can be done by calling the
 	//    Access API on the staked AN.
 
-	block := unittest.BlockFixture()
-
-	proposal := &messages.BlockProposal{
-		Header:  block.Header,
-		Payload: block.Payload,
-	}
+	//block := unittest.BlockFixture()
+	//
+	//proposal := &messages.BlockProposal{
+	//	Header:  block.Header,
+	//	Payload: block.Payload,
+	//}
 
 	// Send block proposal fron consensus node to staked AN
-	suite.conGhost.Send(suite.ctx, engine.PushBlocks, proposal, suite.stakedID)
-
-	m := make(chan interface{})
-	go func() {
-		_, msg, err := suite.unstakedReader.Next()
-		suite.Require().Nil(err, "could not read next message")
-		suite.T().Logf("unstaked ghost recv: %T", msg)
-
-		m <- msg
-	}()
-
-	// Check that the unstaked AN receives the message
-	select {
-	case msg := <-m:
-		suite.Assert().Equal(msg, proposal)
-	case <-time.After(5 * time.Second):
-		suite.T().Fatal("timed out waiting for next message")
-	}
+	//suite.conGhost.Send(suite.ctx, engine.PushBlocks, proposal, suite.stakedID)
+
+	// currently the setup doesn't wait for all the nodes to have completely started
+	time.Sleep(10*time.Second)
+
+	stakedANClient, err := client.New(fmt.Sprintf(":%s", suite.net.AccessPorts[testnet.AccessNodeAPIPort]), grpc.WithInsecure())
+	require.NoError(suite.T(), err)
+
+
+	fmt.Println("pinging the staked AN")
+	ctx := context.Background()
+	err = stakedANClient.Ping(ctx)
+	require.NoError(suite.T(), err)
+
+	unstakedANClient, err := client.New(fmt.Sprintf(":%s", suite.net.AccessPorts[testnet.UnstakedAccessNodeAPIPort]), grpc.WithInsecure())
```

Comment:
Instead of using different Access API ports for staked and unstaked node, why don't we instead have a single `AccessNodeAPIPort` and do `suite.net.ContainerByName("unstaked_access_1").Ports[AccessNodeAPIPort]`